### ISSUE TITLE
proxy freezez on start

### ISSUE THREAD

**Issue by johnsky007** [TelegramMessenger/MTProxy#275]
Hi, I use a ubuntu 18.04 vps, and after doing all steps that described , tried running `root@ajtp:~/MTProxy/objs/bin# ./mtproto-proxy -u root -p 8888 -H 443 -S a01b7a8abbd2bb6b26cdc9f475985bac --aes-pwd proxy-secret proxy-multi.conf -M 1` command I got this and system freezes until terminate it with Ctrl+C, Here in log:
`[3317][2019-03-07 06:33:48.050379 local] Invoking engine mtproxy-0.01 compiled at Mar  7 2019 06:09:17 by gcc 7.3.0 64-bit after commit 2c942119c4ee340c80922ba11d14fb3b10d5e654
[3317][2019-03-07 06:33:48.050708 local] config_filename = 'proxy-multi.conf'
[3317][2019-03-07 06:33:48.051138 local] creating 1 workers
[3317][2019-03-07 06:33:48.053074 local] Started as [198.*.*.*:8888:3317:1551958428]
[3317][2019-03-07 06:33:48.053608 local] configuration file proxy-multi.conf re-read successfully (809 bytes parsed), new configuration active
[3317][2019-03-07 06:33:48.054060 local] main loop
READ: Resource temporarily unavailable
[3318][2019-03-07 06:33:48.056597 local] Started as [198.*.*.*:8888:3318:1551958428]
[3318][2019-03-07 06:33:48.057023 local] configuration file proxy-multi.conf re-read successfully (809 bytes parsed), new configuration active
[3318][2019-03-07 06:33:48.057289 local] main loop`

Any ideas welcome!

**Comment by HirbodBehnam** [comment]
It's not frozen it's actually running! Use `tmux` or `screen` or create a service to make MTproto run on background.
Make sure you open your ports in firewall.

**Comment by johnsky007** [comment]
> It's not frozen it's actually running! Use `tmux` or `screen` or create a service to make MTproto run on background.
> Make sure you open your ports in firewall.

Thanks for reply, but if you notice in log there is a suspicious term: 

> "READ: Resource temporarily unavailable"

, I guess this is a error.

**Comment by HirbodBehnam** [comment]
Sorry my bad
Have you seen #50 ?

**Comment by johnsky007** [comment]
yes, I check that ufw is disabled in my vps, nothing more is available in #50 !
